### ISSUE TITLE
Incorrect warning to use $= even when using $=

### ISSUE THREAD

**Issue by dogben** [PolymerLabs/polylint#25]
When I run polylint on [this file](https://github.com/google/skia-buildbot/blob/master/res/imp/9/paging.html), it complains:
```
paging.html:36:11
    The expression {{ decrement(page) }} bound to the attribute 'data-page$' should use $= instead of =.
paging.html:43:11
    The expression {{ increment(page) }} bound to the attribute 'data-page$' should use $= instead of =.
paging.html:46:11
    The expression {{ decrement(allPages) }} bound to the attribute 'data-page$' should use $= instead of =.
```

Line 36 is `<paper-button on-click="toPageClicked" disabled$="{{ isFirstPage(page) }}" data-page$="{{ decrement(page) }}">` i.e. it is already using $= for data-page.

**Comment by garlicnation** [comment]
Fixed in 2.2.0
